shap/maskers/_tabular.py

```diff
@@ -17,7 +19,7 @@ class Tabular(Masker):
     """ A common base class for Independent and Partition.
     """
 
-    def __init__(self, data, max_samples=100, clustering=None):
+    def __init__(self, data, max_samples=100, clustering=None, impute=None):
```

Comment:
I would add this keyword only for the impute class ant not for tabular. See my comment below.